sys/include/crypto/modes/ccm.h

```diff
@@ -26,11 +27,16 @@
 extern "C" {
 #endif
 
-#define CCM_ERR_INVALID_NONCE_LENGTH -2
-#define CCM_ERR_INVALID_CBC_MAC -3
-#define CCM_ERR_INVALID_DATA_LENGTH -3
-#define CCM_ERR_INVALID_LENGTH_ENCODING -4
-#define CCM_ERR_INVALID_MAC_LENGTH -5
+/**
+ * @name CCM error codes
+ * @{
+ */
+#define CCM_ERR_INVALID_NONCE_LENGTH        (-2)
+#define CCM_ERR_INVALID_CBC_MAC             (-3)
+#define CCM_ERR_INVALID_DATA_LENGTH         (-3)
+#define CCM_ERR_INVALID_LENGTH_ENCODING     (-4)
+#define CCM_ERR_INVALID_MAC_LENGTH          (-5)
+/** @} */
```

Comment:
Unrelated

Reply:
No, fixing the doxygen for this file is allowing the processing of doxygen in this file and, in the end, triggers a warning from doxygen because of undocumented defines. This change is fixing this. Do you want it in a separate commit ?